HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentsController.swift

```diff
@@ -8,28 +8,52 @@
 import UIKit
 import GiniHealthAPILibrary
 
+/**
+ Data structure protocol for the stored invoices/documents needed for Payment Component Controller
+ */
 public protocol GiniDocument {
+    /// The document's unique identifier.
     var documentID: String { get set }
+    /// The document's amount to pay.
     var amountToPay: String? { get set }
+    /// The document's payment due date.
     var paymentDueDate: String? { get set }
+    /// The document's recipient name.
     var recipient: String? { get set }
+    /// Boolean value that indicates if the document is payable. This is obtained by calling the checkIfDocumentIsPayable method.
     var isPayable: Bool? { get set }
+    /// Store payment provider for each document/invoice. Payment provider is obtained by PaymentComponentsController and it's only stored here if the payment provider is installed
     var paymentProvider: PaymentProvider? { get set }
 }
 
+/**
+ Delegate to inform about the current status of the Payment Components Controller.
```

Comment:
[suggestion]: I will rewrite this like:
```
 Protocol used to provide updates on the current status of the Payment Components Controller.
 Uses a callback mechanism to handle payment provider requests.
```

but I'm not sure if the second sentence should be here or in another place. Or should we remove it? For me doesn't make sens here.
Thoughts?